### PR TITLE
Moves redux middleware to actions/utils

### PR DESCRIPTION
**Pull request by lukaszsobek**
Associated Issue: #4537

### Summary of Changes

* created new folder, moved files, deleted old folders
* modified references to moved files

### Test Plan
- ran a text search on all files in repo to see if they reference the moved files
- launched server and watched for error messages

### CODE REVIEW

**Comment by jasonLaster** [comment]
seeing some jest errors: https://circleci.com/gh/devtools-html/debugger.html/8211?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by bomsy** [comment]
@lukaszsobek You might need to update the paths in some of the tests causing the jest errors mentioned by @jasonLaster 

image

File: src/actions/expressions.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
   getSelectedFrameId,
   getSource
 } from "../selectors";
-import { PROMISE } from "../utils/redux/middleware/promise";
+import { PROMISE } from "./utils/promise";
```
Comment (jasonLaster) [inline src/actions/expressions.js]: hmm, i still like having the middleware directory so it is clear what it is:

`"./utils/middleware/promise";`

i agree redux is redundant

File: src/actions/utils/middleware/create-store.js [hunk 2]
```diff
@@ -11,12 +11,12 @@
  */
 
 import { createStore, applyMiddleware } from "redux";
-import { waitUntilService } from "./redux/middleware/wait-service";
-import { log } from "./redux/middleware/log";
-import { history } from "./redux/middleware/history";
-import { promise } from "./redux/middleware/promise";
-import { thunk } from "./redux/middleware/thunk";
-import { timing } from "./redux/middleware/timing";
+import { waitUntilService } from "./wait-service";
```
Comment (jasonLaster) [inline src/actions/utils/middleware/create-store.js]: Hmm, i think `createStore` should be in `src/actions/utils`

**Comment by jasonLaster** [comment]
Thanks!

**Pull request merged**